app/assets/stylesheets/spotlight/_header.scss

```diff
@@ -31,112 +28,93 @@ $masthead-image-blur: 1px;
   }
 }
 
-#exhibit-masthead,
-#exhibit-navbar {
-  z-index: 1;
-}
-
-#exhibit-navbar {
-  z-index: 100;
-
-  &.page-masthead {
-    @include transparent-masthead-navigation-menu();
-    background-color: $navbar-transparent-page-bg;
-    border-bottom: 1px solid $navbar-transparent-border;
-    margin-bottom: 0;
-    margin-top: 0;
-
-    .navbar-nav {
-      float: right;
-    }
-  }
-
-  .navbar-form {
-    text-align: right;
-  }
-}
-
-#exhibit-masthead.with-image + #exhibit-navbar.navbar-default {
-  @include transparent-masthead-navigation-menu();
-  background-color: $navbar-transparent-bg;
-  border-top: 1px solid $navbar-transparent-border;
-}
-
-.navbar + .navbar {
-  margin-top: 0;
-}
-
 #header-navbar {
   margin-bottom: 0;
 }
 
-#exhibit-masthead {
-  height: $masthead-height;
-  margin-bottom: -50px;
+.masthead {
+  // allow dropdowns in the top nav to stack above the masthead
+  z-index: $zindex-navbar - 1;
```

Comment:
Properties should be ordered background-color, border-bottom, margin-bottom, overflow, padding, position, z-index